components/component_page/avatar.html

```diff
@@ -1,12 +1,90 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="vidhi thakur" />
+    <link rel="stylesheet" href="../component_style/avatar.css" />
+    <link rel="stylesheet" href="../../styles/variable.css" />
     <title>Avatar | Savvy UI</title>
-</head>
-<body>
-    <h1>Avatar rendered here</h1>
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <h3>avatars with images</h3>
+    <section class="avatar-box">
+      <div class="avatar avatar-xs-size">
+        <img
+          src="https://avatars.dicebear.com/api/adventurer/vidhi97.svg"
+          alt="avatar"
+        />
```

Comment:
avatar-xs  (same for others lg ,xlg,s )would be fine ,someone mentioned it in the code review session,like we can undestand from avatar-xs only :),so always try to keep the class names shorter.